### PR TITLE
VPA - Update docs a little

### PR DESCRIPTION
**Pull request by adrianmoisey**
Mostly markdown linting things, but some small fixes (I found some unnessesary whitespace).

I made one wording change to try put emphasis on the fact that the VPA's primary purpose is resource requests and not limits

#### What type of PR is this?

/kind cleanup
/kind documentation

#### What this PR does / why we need it:

The docs for the VPA need to be cleaned up a bunch, I gave it a once over to try make things neater, and ended up mostly on small lifting type issues.
I also tried to make the intro paragraph focus on requests, as many users seem to think the VPA controls limits. 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: vertical-pod-autoscaler/README.md [hunk 1]
```diff
@@ -240,35 +243,39 @@ spec:
...
+```console
 kubectl get customresourcedefinition | grep verticalpodautoscalers
 ```
 
 ### Components of VPA
 
 The project consists of 3 components:
 
-* [Recommender](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/recommender/README.md) - it monitors the current and past resource consumption and, based on it,
+- [Recommender](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/recommender/README.md) - it monitors the current and past resource consumption and, based on it,
```
Comment (omerap12) [inline vertical-pod-autoscaler/README.md:272]: Can we do:
[Recommender](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/recommender/README.md) - monitors the current and past resource consumption and, based on it,

File: vertical-pod-autoscaler/README.md [hunk 2]
```diff
@@ -240,35 +243,39 @@ spec:
...
   provides recommended values for the containers' cpu and memory requests.
 
-* [Updater](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/updater/README.md) - it checks which of the managed pods have correct resources set and, if not,
+- [Updater](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/updater/README.md) - it checks which of the managed pods have correct resources set and, if not,
```
Comment (omerap12) [inline vertical-pod-autoscaler/README.md:275]: Same here

File: vertical-pod-autoscaler/README.md [hunk 3]
```diff
@@ -240,35 +243,39 @@ spec:
...
   kills them so that they can be recreated by their controllers with the updated requests.
 
-* [Admission Plugin](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/admission-controller/README.md) - it sets the correct resource requests on new pods (either just created
+- [Admission Plugin](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/admission-controller/README.md) - it sets the correct resource requests on new pods (either just created
```
Comment (omerap12) [inline vertical-pod-autoscaler/README.md:278]: Same here

**Review by omerap12 — COMMENTED** [review]
Overall, it looks great! Just a small note from my side.  
Thank you for handling this! :)

File: vertical-pod-autoscaler/README.md (hunk 1 above)
Comment (adrianmoisey) [reply vertical-pod-autoscaler/README.md]: Good idea, fixed in 586813cbb5

**Review by kwiesmueller — APPROVED** [review]
/lgtm
/approve

**Pull request merged**